bin/hermit.hcl

```diff
@@ -5,6 +5,7 @@ env = {
 
   // Python:
   "LANG": "en_US.UTF-8",
```

Comment:
From @alcuadrado ([comment](https://github.com/NomicFoundation/slang/pull/1006#issuecomment-2161217358)):

> Maybe a less disruptive solution can be adding `TZ=UTC` to the `mkdocs` command in here:

Given that the error was raised from the common std library `python3.11/zoneinfo/_tzpath.py`, I think setting it here for all modules would fix the issue for all other modules/commands, as any of them could also be using `zoneinfo`?

Reply:
I don't really know how often `infra` uses python, so I don't understand which solution is better. I was just trying to actually find the root of the problem when I saw that alternative.